### ISSUE TITLE
link authorea paper with github repo?

### ISSUE THREAD

**Issue by epruesse** [bioconda/bioconda-paper#12]
Apparently Authorea can be made to link up with a github repo (syncing both ways):

https://intercom.help/authorea/user-settings/user-settings/syncing-articles-to-github

@johanneskoester Thoughts? It may be worthwhile to do even if it's late now, if only to preserve the paper as we preserve everything else. Just close this if you decide it's not worth the bother.

**Comment by epruesse** [comment]
Hmm. Looks like you got there half-way already. The target repo doesn't seem to exist though.

**Comment by johanneskoester** [comment]
Yes, I will look into this again. I tried it before and it was a bit cumbersome. In any case, I will do it after submission, just to be sure that I do not destroy something last minute. It is noticeable that the authorea is still beta. But I find their responsiveness on bug reports and feature requests really outstanding.
